Add privacy policy (#825)
grote/Transportr: privacy-policy.md

# Privacy Policy

> The Transportr developers do not collect any data themselves. But by using the app several personnal informations might get send to third party providers:

Reviewer: ```suggestion
The Transportr developers do not collect any data themselves. But by using the app some personal information might get send to third party providers:
```

- your location
- your IP address
- trip details (starting and ending point, date and time of travel...)
- the fact you are using Transportr

See down below for more detailed explanations.

## Location usage

Your location is only retrieved for the following 2 usages:

- retrieve the map tiles around your location from the Mapbox API,
- use the location as a start or end of a trip query.

## Data usage

### Transport information providers

> When searching for a trip, the entered data (trip details (starting and ending point, date and time of travel...), your IP address and the fact you are using Transportr) will be available to the local transport provider you selected. For more information about how the selected transport provider stores your data, check their privacy policies.

Reviewer: ```suggestion
When searching for a trip, the entered data (trip details, your IP address and the fact you are using Transportr) will be available to the local transport provider you selected. For more information about how the selected transport provider stores your data, check their privacy policies.
```

### Map tiles

> The map tiles are retrived from the Mapbox API. The data they receive about you when dowloading the map is your IP address, your location and the fact you are using Transportr. For more information about how the selected transport provider stores your data, check their [privacy policy](https://www.mapbox.com/legal/privacy).

Reviewer: ```suggestion
The map tiles are retrieved from the Mapbox API. The data they receive about you when downloading the map is your IP address, the location for which you download the map (which might or not be your current location) and the fact you are using Transportr. For more information about how Mapbox uses your data, check their [privacy policy](https://www.mapbox.com/legal/privacy).
```
